src/pages/solution/adobe-analytics/migrate-to-edge-network.md

```diff
@@ -22,7 +22,7 @@ Edge Bridge only supports the Analytics drop-in migration case. If you are looki
 |       |  API for sending data | Extensions| XDM transform location |  Platform support with latest Mobile SDKs | Can send to other services (configurable in Datastreams)  |
 | ----------- | ----------- |-----------|-----------|-----------|-----------|
 | **Edge Network extension** | Edge.sendEvent | 1. Edge Network extension <br/> 2. Identity for Edge Network extension | Client-side| iOS, Android, tvOS, Flutter, React Native | Yes |
-| **Edge Bridge extension** | MobileCore.trackAction <br/> MobileCore.trackState | 1. Edge Bridge extension <br/> 2. Edge Network extension <br/> 3. Identity for Edge Network extension| Server-side through Data Prep mapping| iOS, Android, tvOS, Flutter, React Native | Yes |
+| **Edge Bridge extension** | MobileCore.trackAction <br/> MobileCore.trackState | 1. Edge Bridge extension <br/> 2. Edge Network extension <br/> 3. Identity for Edge Network extension| Not required for Analytics. <br/> Server-side Data Prep mapping if XDM needed for other services. | iOS, Android, Flutter, React Native | Yes |
```

Comment:
Not required for Analytics. <br/> Server-side Data Prep mapping in datastream if XDM is needed for other services.